.github/workflows/run_test.yaml

```diff
@@ -70,12 +70,12 @@ jobs:
         run: pdm fmt-check
       - name: Test with pytest
         run: |
-          docker-compose -f tests/docker-compose.yaml up -d
+          docker compose -f tests/docker-compose.yaml up -d
           sleep 20
           pdm test
       - name: Test SSL connection with pytest
         run: |
-          enable_ssl=true docker-compose -f tests/docker-compose-ssl.yaml up -d
+          enable_ssl=true docker compose -f tests/docker-compose-ssl.yaml up -d
```

Comment:
latest gh action runner comes with a different version of docker compose, it failed here with `docker-compose`, this fixed it. ref: https://askubuntu.com/questions/1508129/docker-compose-giving-containerconfig-errors-after-update-today